### PR TITLE
Bug 78/v2

### PR DESCRIPTION
**Pull request by shivan1b**
Closes redmine ticket: https://redmine.openinfosecfoundation.org/issues/3055

### CODE REVIEW

**Comment by catenacyber** [comment]
cf https://redmine.openinfosecfoundation.org/issues/78

**Comment by jasonish** [comment]
This test is failing for me on master? Does it require a PR or fix?

**Comment by catenacyber** [comment]
Apparently it just needs to remove one line in the test.yaml file :

```
diff --git a/tests/bug-78-uricontent/test.yaml b/tests/bug-78-uricontent/test.yaml
index cc76902..98a88ea 100644
--- a/tests/bug-78-uricontent/test.yaml
+++ b/tests/bug-78-uricontent/test.yaml
@@ -85,7 +85,6 @@ checks:
         protocol: HTTP/1.0
         status: 404
         url: /blah/
-      pcap_cnt: 7
       proto: TCP
       src_ip: 208.69.36.231
       src_port: 80
```

File: tests/bug-78-http-uri/test.yaml [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+      http:
+        hostname: www.google.com
+        http_content_type: text/html
+        http_method: GET
+        http_user_agent: Wget/1.11.4
+        length: 1194
+        protocol: HTTP/1.0
+        status: 404
+        url: /blah/
+      pcap_cnt: 7
```
Comment (catenacyber) [inline tests/bug-78-http-uri/test.yaml:59]: Can you remove this test ?

Comment (inashivb) [reply tests/bug-78-http-uri/test.yaml:59]: Please check #376

**Comment by inashivb** [comment]
Replaced with #376

**Pull request closed without merging**